frigate/ffmpeg_presets.py

```diff
@@ -43,7 +43,10 @@ def get_selected_gpu(self) -> str:
         return ""
 
 
-TIMEOUT_PARAM = "-timeout" if os.path.exists(BTBN_PATH) else "-stimeout"
+LIBAVFORMAT_VERSION_MAJOR = int(
```

Comment:
I'm concerned that this will get run many times, I think we may need to do something like the LibvaGpuSelector so the value of this is cached because we do not want the overhead of spawning this process every time it is imported.

Reply:
Oh, this is imported more times than I realized. Will do.

Reply:
@madsciencetist actually, I think it might be easier to have s6 check the ffmpeg version before starting frigate and save that to an env variable.